wolfssl/openssl/ssl.h

```diff
@@ -318,7 +318,7 @@ typedef STACK_OF(ACCESS_DESCRIPTION) AUTHORITY_INFO_ACCESS;
 #define SSL_verify_client_post_handshake wolfSSL_verify_client_post_handshake
 #define SSL_set_post_handshake_auth     wolfSSL_set_post_handshake_auth
 #define SSL_CTX_set_post_handshake_auth wolfSSL_CTX_set_post_handshake_auth
-#define SSL_pending                     wolfSSL_pending
+#define SSL_pending(x)                  wolfSSL_pending((WOLFSSL*)x)
```

Comment:
Oh, technically the x should be wrapped in (). `wolfSSL_pending((WOLFSSL*)(x))`. Can you please update these?

Reply:
Done